content/components/relative-time.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Relative time
+status: Experimental
+---
+
+import {Box, Text} from '@primer/react'
+
+<Box sx={{fontSize: 3}} class="lead" as="p">
+  A relative time displays the relative time difference between the source date-time and the provided date-time.
+</Box>
+
+## Content
+
+### Past
+
+For past dates, the relative time outputs different formats depending on the time difference.
+
+Within the current month, the output shows a relative date in the format of `X seconds/minutes/hours/days ago`. If the time difference is less than 1 minute, it shows `just now`.
+
+After a month has passed, the output shows a relative date in the format `on D MMM` or `on D MMM, YYYY` depending on the current year.
+
+<img
+  width="960"
+  src="https://user-images.githubusercontent.com/912236/192233477-8c30b28b-5f48-44c6-b373-39a482f745a1.png"
+  role="presentation"
+/>
+<Caption>A list of items using the past relative outputs.</Caption>
+
+### Future
+
+As well as past dates, future dates output different formats depending on the time difference.
+
+If the future date is due to happen within the current month, the relative time outputs the format of `in X seconds/minutes/hours/days`. If the time difference is less than 1 minute, it shows `just now`.
+
+If the future date is due to happen within the following months, the relative time outputs the format of `on D MMM` or `on D MMM, YYYY` when following years.
```

Comment:
```suggestion
If the date in the future but not within the current month, the format `on D MMM` is used. If the date is in the next year or further, the format `on D MMM, YYYY` is used.
```

Reply:
I think that `If the date is in the future` might be redundant as we are in the `Future date` section. What about this?

```suggestion
If the date is in the following months, the format `on D MMM` is used. If the date is in the next year or further, the format `on D MMM, YYYY` is used.
```

Reply:
Great 👍